scss/civicrm/administer/civi-case/_case-types.scss

```diff
@@ -44,7 +49,8 @@
 
   #crm-main-content-wrapper {
     .crm-submit-buttons {
-      border-top: none;
+      background: none !important;
```

Comment:
what's this for? I don't see any mention of button background in the PR or in the screenshot